pfrl/wrappers/monitor.py

```diff
@@ -1,8 +1,22 @@
 import time
 from logging import getLogger
 
-from gym.wrappers import Monitor as _GymMonitor
-from gym.wrappers.monitoring.stats_recorder import StatsRecorder as _GymStatsRecorder
+try:
+    from gym.wrappers import Monitor as _GymMonitor
+except ImportError:
+
+    class _Stub:
+        def __init__(self, *args, **kwargs):
+            raise RuntimeError("Monitor is not available in this version of gym")
+
+    class _GymMonitor(_Stub):  # type: ignore
+        pass
+
+    class _GymStatsRecorder(_Stub):
+        pass
+
+else:
+    from gym.wrappers.monitoring.stats_recorder import StatsRecorder as _GymStatsRecorder  # type: ignore, isort: skip  # noqa: E501
```

Comment:
Sorry, I haven't checked mypy for the previous suggestion, but `# type: ignore, isort: skip` seems not working for mypy, causing another error in CI. https://ci.preferred.jp/pfrl.lint/96657/#L52
```suggestion
    from gym.wrappers.monitoring.stats_recorder import StatsRecorder as _GymStatsRecorder  # type: ignore  # isort: skip  # noqa: E501
```
I confirmed this now pass black, isort, flake8, and mypy.